startup_scripts/startup_script_utils/permissions.py

```diff
@@ -0,0 +1,18 @@
+from django.contrib.auth.models import Permission
+
+
+def set_perms_from_yaml(subject=None, permissions=None):
```

Comment:
Personally I'm not happy with the name. The first reason is the abbreviation `perms`, which could just be written as `permissions`. The second reason is the `from_yaml` part. As I see it, `permissions` is just a list of strings, there's not yaml in there. And `subject` is has nothing to do with yaml either.

Also, if either of the arguments must not be null, why bother setting default parameters at all?

```suggestion
def set_permissions(subject, permissions):
```